### PR TITLE
Product Recommendations release notes for 3.2.0

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) updates the release notes for product recommendations 3.2.0.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/release-notes.html

### CODE REVIEW

File: src/_data/mde.yml [hunk 1]
```diff
@@ -310,7 +310,7 @@ extensions:
     name: Product Recommendations
     versions:
       -
-        name: 3.10.0
+        name: 3.2.0
```
Comment (misha-kotov) [inline src/_data/mde.yml:313]:
```suggestion
        name: 3.2.1
```

File: src/recommendations/release-notes.md [hunk 2]
```diff
@@ -8,13 +8,23 @@ The release notes contain updates to the following Product Recommendations modul
...
+-  Visual similarity recommendation type support for Product Recommendations (optional) module: `magento/module-visual-product-recommendations`
 
 The release notes include:
 
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## 3.2.0 of magento/product-recommendations for Magento Commerce 2.4.0
+
+-  {:.new}Added the ability to [resync your catalog to SaaS services from the Admin](https://docs.magento.com/user-guide/system/catalog-sync.html)
```
Comment (misha-kotov) [inline src/recommendations/release-notes.md:20]:
```suggestion
-  {:.new}Added the ability to [monitor and manage](https://docs.magento.com/user-guide/system/catalog-sync.html) the catalog sync process from the Admin Panel. 
```

File: src/recommendations/release-notes.md [hunk 3]
```diff
@@ -8,13 +8,23 @@ The release notes contain updates to the following Product Recommendations modul
...
+-  {:.new}Added the ability to specify [inclusion and exclusion filters](https://docs.magento.com/user-guide/marketing/recommendation-incl-excl.html) for your recommendations.
```
Comment (misha-kotov) [inline src/recommendations/release-notes.md:21]:
```suggestion
-  {:.new}Added [inclusion and exclusion filters](https://docs.magento.com/user-guide/marketing/recommendation-incl-excl.html) to control what products can and cannot show up in recommendations.
```

**Review by misha-kotov — CHANGES_REQUESTED** [review]

File: src/recommendations/release-notes.md [hunk 4]
```diff
@@ -8,13 +8,23 @@ The release notes contain updates to the following Product Recommendations modul
...
+-  {:.new}Added the [Visual similarity recommendation type](https://docs.magento.com/user-guide/marketing/prex-type-visualsim.html)
+
+## 1.2.0 of magento/module-page-builder-product-recommendations for Page Builder for Magento Commerce 2.4.0 and later
```
Comment (misha-kotov) [inline src/recommendations/release-notes.md:24]:
```suggestion
## 1.2.1 of magento/module-page-builder-product-recommendations for Page Builder for Magento Commerce 2.4.0 and later
```

**Review by keharper — APPROVED** [review]

**Comment by erikmarr** [comment]
running tests

**Pull request merged**